src/main/java/use_case/student_homepage/like/StudentLikeInteractor.java

```diff
@@ -53,7 +60,8 @@ public void execute(StudentLikeInputData studentLikeInputData) {
             }
             postData.put("liked", !(Boolean) postData.get("liked"));
 
-            final StudentLikeOutputData studentLikeOutputData = new StudentLikeOutputData(postData, currClub.getUsername());
+            final StudentLikeOutputData studentLikeOutputData = new StudentLikeOutputData(postData,
+                    currClub.getUsername());
             studentLikePresenter.prepareSuccessView(studentLikeOutputData);
         }
     }
```

Comment:
well done refactoring my part, the logic seems correct as far as I'm aware.

Reply:
Thank!